content/python/concepts/strings/terms/ljust/ljust.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.ljust()'
+Description: 'Left-aligns a string with a specified fill character'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.ljust()`** method left-aligns a string with a specified fill character
```

Comment:
Please add the following points:
- is a method for strings
- returns a left0aligned string of the length given
Feel free to cover other relevant information.